np.Tidy/src/tidyHelpers.js

```diff
@@ -33,6 +33,8 @@ export type TidyConfig = {
   runRemoveSectionFromNotesCommand: boolean,
   removeTriggersFromRecentCalendarNotes: boolean,
   removeTodayTagsFromCompletedTodos: boolean,
+  moveTopLevelTasksInEditor: boolean,
+  moveTopLevelTasksInEditor: string,
```

Comment:
This looks like duplication of setting names with different types. Is that a typo?